test/system/030-run.bats

```diff
@@ -522,4 +522,9 @@ json-file | f
     run_podman untag $IMAGE $newtag $newtag2
 }
 
+@test "podman run with --net=host and --port prints warning" {
+    run_podman run -d --rm -p 8080 --net=host $IMAGE ls > /dev/null
```

Comment:
The `>/dev/null` here is a NOP. Not worth resubmitting for, though. Just an FYI for next time. (All output is captured in `$output`)